dashboard/db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
Looks intentional. From the [changelog](https://github.com/rails/rails/blob/5-0-stable/activerecord/CHANGELOG.md#rails-500-june-30-2016):

> Omit default limit values in dumped schema. It's tidier, and if the defaults change in the future, we can address that via Migration API Versioning.